viewer-crd-controller/tox.ini

```diff
@@ -3,6 +3,7 @@
 [tox]
 skipsdist = True
 skip_missing_interpreters = True
+envlist = unit, sanity, integration
```

Comment:
In your file you've got: `integration, sanity, export-to-docker, pack`. I think You want to update this envlist.

Reply:
Updated! I 'm wondering how `tox` uses those, since it's working without those configured properly.

Reply:
Got it 
> `envlist` is a list of all environments that would be run if you invoke tox without the -e option.